process/interceptors/multiDataInterceptor.go

```diff
@@ -18,6 +18,7 @@ type MultiDataInterceptor struct {
 	factory     process.InterceptedDataFactory
 	processor   process.InterceptorProcessor
 	throttler   process.InterceptorThrottler
+	isForMe     process.InterceptedDataVerifier
```

Comment:
isForMe sounds like a bool variable, maybe change this to dataVerifier?

Reply:
done